tensorflow_recommenders_addons/dynamic_embedding/python/ops/dynamic_embedding_variable.py

```diff
@@ -100,6 +100,23 @@ def default_partition_fn(keys, shard_num):
   return ids
 
 
+class DeGraphKeys(object):
```

Comment:
de.GraphKeys maybe better, de.DeGraphKeys is a bit redundant

Reply:
fixed

Reply:
@googlebot I fixed it.